pkg/meta/annotations.go

```diff
@@ -197,6 +197,15 @@ const (
 	NMAHealthProbeInitialDelaySeconds = "initial-delay-seconds"
 	//
 	// Use GenNMAHealthProbeAnnotationName to generate the name.
+
+	// Set of annotations for scrutinize
+	//
+	ScrutinizePodTimeToLive             = "vertica.com/scrutinize-pod-ttl"
+	ScrutinizePodTimeToLiveDefaultValue = 1800 // 30min
+	ScrutinizePodRestartPolicy          = "vertica.com/scrutinize-pod-restart-policy"
```

Comment:
Lets add a separate comment for this one too.
```suggestion
        // Allows you to control the restartPolicy of the scrutinize pod.
	ScrutinizePodRestartPolicyAnnotation         = "vertica.com/scrutinize-pod-restart-policy"
```